src/app/models/project.ts

```diff
@@ -47,6 +47,7 @@ export class Project {
   substitution: Boolean;
   updatedBy: String;
   isPublished: boolean;
+  collectionNotice: String;
```

Comment:
I know there are some conflicting entries here among the other types but you actually want `string` here. The difference is that `String` refers to the class and `string` refers to the primitive in JS, which is more accurate to how you're trying to type this property.


```suggestion
  collectionNotice: string;
```

https://stackoverflow.com/a/34993650